src/__tests__/is-ios-webview.ts

```diff
@@ -36,6 +38,34 @@ describe("isIosWebview", () => {
   it("returns false for desktop Safari", () => {
     expect(isIosWebview(AGENTS.macSafari7_0_2)).toBe(false);
   });
+  
+  it("returns true when supported and not case-sensitive", () => {
```

Comment:
This actually is likely a red-herring based on the reported bugs we had. But, honestly probably worth keeping to ensure that case is covered.